src/core/utils/pdf/pdf.ts

```diff
@@ -0,0 +1,69 @@
+import * as pdfjsLib from 'pdfjs-dist';
+
+pdfjsLib.GlobalWorkerOptions.workerSrc = `pdf.worker.min.mjs`;
```

Comment:
Can we add a hash and version to the file name?

`pdf.worker.min-{version}-{short hash of file}.mjs`